pkg/maglev/maglev.go

```diff
@@ -74,33 +74,58 @@ func getPermutation(backends []string, m uint64) []uint64 {
 	return perm
 }
 
+//BackendPoint is a backend point with weight
```

Comment:
Could you add a space like so to stay consistent with the rest of the file? There are many occurrences of this. 

```suggestion
// BackendPoint is a backend point with weight
```

Reply:
done !